client/delete_lease.go

```diff
@@ -0,0 +1,30 @@
+package client
+
+import (
+	"io/ioutil"
+	"log"
+	"net/http"
+
+	"github.com/deis/k8s-claimer/htp"
+)
+
+// DeleteLease deletes a lease
+func DeleteLease(server, authToken, leaseToken string) error {
+	endpt := newEndpoint(htp.Delete, server, "lease/"+leaseToken)
+	resp, err := endpt.executeReq(getHTTPClient(), nil, authToken)
+	if err != nil {
+		log.Fatalf("Error executing %s (%s)", endpt, err)
+		return errHTTPRequest{endpoint: endpt.String(), err: err}
+	}
+	defer resp.Body.Close()
+	if resp.StatusCode != http.StatusOK {
+		bodyBytes, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			bodyBytes = nil
+			return err
+		}
+		log.Fatalf("Error deleting. Code: %d, Body: %s", resp.StatusCode, string(bodyBytes))
```

Comment:
I think this line moved with the rest of this code from the cli's commands package and I accidentally did fix this. I will.